src/__tests__/Search.test.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("Search", () => {
+  const validProps = {
+    setSearchResults: jest.fn()
+  }
+
+  it("renders correctly", () => {
+    const { asFragment } = render(<Search setSearchResults={validProps.setSearchResults} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("button displays the correct text", () => {
+    render(<Search setSearchResults={validProps.setSearchResults} />)
+
+    const button = screen.getAllByRole("button");
```

Comment:
Id be tempted to group these together as really they are still part of the setup of the test:
```
    render(<Search setSearchResults={validProps.setSearchResults} />)
    const button = screen.getAllByRole("button");
```